corehq/apps/domain/views/settings.py

```diff
@@ -584,6 +587,12 @@ def post(self, request, *args, **kwargs):
         user = User.objects.get(pk=uid)
         couch_user = CouchUser.from_django_user(user)
         clear_login_attempts(couch_user)
+        domain = None
+        if couch_user.is_commcare_user():
+            domain = couch_user.domain
```

Comment:
style super nit `domain = couch_user.domain if couch_user.is_commcare_user() else None` I think this is equivalent for brevity

Reply:
done! [3b4201d](https://github.com/dimagi/commcare-hq/pull/36676/commits/3b4201d44336c738ac8c783f394af0236bf8a4c7)